Devices.Infrastructure/Enmeshed.BuildingBlocks.Infrastructure/EventBus/GoogleCloudPubSub/DefaultGoogleCloudPubSubPersisterConnection.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Google.Api.Gax;
+using Google.Apis.Auth.OAuth2;
+using Google.Cloud.PubSub.V1;
+using Grpc.Core;
+
+namespace Devices.Infrastructure.Enmeshed.BuildingBlocks.Infrastructure.EventBus.GoogleCloudPubSub
+{
+    public class DefaultGoogleCloudPubSubPersisterConnection : IGoogleCloudPubSubPersisterConnection
+    {
+        private const string MESSAGE_ATTRIBUTE_PREFIX = "Subject";
+
+        private readonly string _projectId;
+        private readonly TopicName _topicName;
+        private readonly string _subscriptionNamePrefix;
+        private readonly GoogleCredential? _gcpCredentials;
```

Comment:
```suggestion
        private readonly GoogleCredential _gcpCredentials;
```

We don't have nullable reference types enabled in this project.